docs/component_conventions.md

```diff
@@ -143,7 +174,7 @@ With the exception of namespaces, follow the [GOV.UK Frontend CSS conventions](h
 
 `.block__element--modifier {}`
 
-All CSS selectors should follow the BEM naming convention shown above, explained in [more detail here](https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/coding-standards/css.md#block-element-modifier-bem).
+All CSS selectors should follow the BEM naming convention shown above, explained in [more detail here](https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/coding-standards/css.md#block-element-modifier-bem). Try to avoid nesting classes if possible.
```

Comment:
I think the addition here about nesting repeats what is said in the paragraph immediately below?

Reply:
Not exactly. I'm trying to say we don't need to have long chains of selectors, like `.class1 .class2 .class3 .class4 { ... }` whereas the para below is talking about not naming individual classes to reflect the dom structure, e.g. `.block__elem1__elem2__elem3`. Trying to think of a way to make this clearer.